Source/santad/Logs/EndpointSecurity/Serializers/Utilities.mm

```diff
@@ -125,14 +124,44 @@ const mach_port_t GetDefaultIOKitCommsPort() {
   IORegistryEntryCreateCFProperties(device, &device_properties, kCFAllocatorDefault, kNilOptions);
   NSDictionary *properties = CFBridgingRelease(device_properties);
   IOObjectRelease(device);
+  return properties;
+}
+
+NSString *DiskImageForDevice(NSString *devPath) {
+  NSString *result;
+
+  // First try getting the image-path property of the parent path of
+  // the given device path. Since at least macOS 13, this mainly works
```

Comment:
```suggestion
  // First try getting the image-path property of the parent device of
  // the given device path. Since at least macOS 13, this mainly works
```